README.md

```diff
@@ -7,6 +7,17 @@
 
 The C++ [OpenTelemetry](https://opentelemetry.io/) client.
 
+## Supported C++ Versions
+
+Code shipped from this repository generally supports the following versions of
+C++ standards:
+
+* ISO/IEC 14882:2011 (C++11, C++0x)
+* ISO/IEC 14882:2014 (C++14, C++1y)
+* ISO/IEC 14882:2017 (C++17, C++1z)
```

Comment:
nit: how about c++20 which just comes out?

Reply:
It is not there yet, we can add it once it is becoming clear.